src/main/drivers/pwm_output_dshot_hal.c

```diff
@@ -148,13 +153,13 @@ FAST_CODE void pwmCompleteDshotMotorUpdate(void)
     }
 }
 
-static void motor_DMA_IRQHandler(dmaChannelDescriptor_t* descriptor)
+FAST_CODE static void motor_DMA_IRQHandler(dmaChannelDescriptor_t* descriptor)
 {
     if (DMA_GET_FLAG_STATUS(descriptor, DMA_IT_TCIF)) {
         motorDmaOutput_t * const motor = &dmaMotors[descriptor->userParam];
         if (!motor->isInput) {
 #ifdef USE_DSHOT_TELEMETRY
-            uint32_t irqStartUs = micros();
+            dshotDMAHandlerCycleCounters.irqAt = SysTick->VAL;
```

Comment:
Maybe DWT->CYCCNT for consistency ?

Reply:
yes, i initially used SysTick->VAL but forgot to update this one to use DWT.  Good spot.